src/ocm/components/clusterConfiguration/CpuArchitectureDropdown.tsx

```diff
@@ -44,43 +58,53 @@ const getInvalidCombinationReason = (
     : undefined;
 };
 
-const CpuArchitectureDropdown = () => {
+type CpuArchitectureDropdownProps = {
+  openshiftVersion: Cluster['openshiftVersion'];
+  day1CpuArchitecture?: CpuArchitecture;
+  cpuArchitectures: SupportLevels;
+};
+
+const CpuArchitectureDropdown = ({
+  openshiftVersion,
+  day1CpuArchitecture,
+  cpuArchitectures,
+}: CpuArchitectureDropdownProps) => {
   const [field, { value: selectedCpuArchitecture }, { setValue }] =
     useField<SupportedCpuArchitecture>(INPUT_NAME);
-  const {
-    values: { openshiftVersion },
-  } = useFormikContext<ClusterCreateParams>();
+
   const [isOpen, setOpen] = React.useState(false);
 
   const prevVersionRef = React.useRef(openshiftVersion);
   const featureSupportLevels = useFeatureSupportLevel();
+  const [currentCpuArch, setCurrentCpuArch] = React.useState<string>(
+    day1CpuArchitecture
+      ? (architectureData[day1CpuArchitecture] as CpuArchitectureItem).label
+      : CpuArchitecture.x86,
+  );
 
+  const isMultiArchSupported = useFeature('ASSISTED_INSTALLER_MULTIARCH_SUPPORTED');
   const enabledItems = React.useMemo(() => {
-    return getSupportedCpuArchitectures().map((cpuArch) => {
-      const disabledReason = getInvalidCombinationReason(
-        featureSupportLevels,
-        cpuArch,
-        openshiftVersion,
-      );
-      return (
-        <DropdownItem
-          key={cpuArch}
-          id={cpuArch}
-          description={architectureData[cpuArch].description}
-          tooltip={disabledReason ? <p>{disabledReason}</p> : undefined}
-          isAriaDisabled={!!disabledReason}
-        >
-          {cpuArch}
-        </DropdownItem>
-      );
-    });
-  }, [featureSupportLevels, openshiftVersion]);
+    return getNewSupportedCpuArchitectures(cpuArchitectures, isMultiArchSupported).map(
+      (cpuArch) => {
+        return (
+          <DropdownItem
```

Comment:
There is a change here that will break some e2e test: [example](https://gitlab.cee.redhat.com/ocp-edge-qe/kni-assisted-installer-auto/-/blob/master/ui_tests/cypress/integration/wizard/validations/clusterDetails.spec.js#L23)

Up to now, for ARM we were showing it as disabled when the OpenShift version didn't support it with a tooltip and an explanation. Do we want to remove this?

Reply:
I've asked to Nir.